### ISSUE TITLE
Auto page reload on network state change or http error

### ISSUE THREAD

**Issue by readandwrite864** [thanksmister/wallpanel-android#212]
**The problem**
Sometimes when device has lost connection, or server is down, WebView shows corresponding error page. When connection established again, nothing actually happens, page doesn't reload. 

**The solution I'd like**
It would be nice to have appropriate setting that for example constantly reloads page after given timeout when error occurs

**Alternatives I've considered**
One can load offline .html wrapper instead, where actual page sits inside <iframe>. Also that wrapper should have some kind of js watcher that will reload page when connection restores. You maybe can also send notify service messages to Wallpanel app from server, but that's also not the case you can consider every time

**Comment by thanksmister** [comment]
Yeah, we would need to get the error from the web page and then reload the web page.  We do know when a MQTT connection is lost and reconnected, but not sure if calling to refresh web page on this event would be good solution.

**Comment by readandwrite864** [comment]
There are cases when MQTT protocol doesn't even used, the app is just for showing web page fullscreen and enshuring that the page will load 100% when possible

**Comment by thanksmister** [comment]
@readandwrite864 If the app were just for displaying web pages, you wouldn't need the app, you can just use Chrome.  MQTT, HTTP, screensaver, motion, camera streams, these are the features.   If we were just using chrome and the page didn't load, you would have to manually refresh.  I think using MQTT to reload the page remotely is a good start.  However, if you want to submit a PR for the WebView to handle errors and reload, I am open for that.

**Comment by R5fan** [comment]
wallpanel actually freezes/crashes on my 2 tablets since a recent update whenever wifi connection is lost. I can easily reproduce it, unplug or reboot my router, and usually both tablets will, after a minute or so,  show android's "app not responding" and they dont recover.  They are very different tablets (one is android 9 I think, the other 10). 
It started happening when I upgraded a week or so ago to the latest version which supports having wallpanel act as a home screen. Enabling or disabling that doesnt seem to make a difference.

**Comment by thanksmister** [comment]
I will try to test this, maybe others can verity this behavior if it’s easy to reproduce.

**Comment by thanksmister** [comment]
This issue should Neón it’s own ticket since it’s unrelated to the this ticket.

**Comment by thanksmister** [comment]
We can try this with https://github.com/thanksmister/wallpanel-android/releases/tag/v0.9.4.9.  But the app can reload the page only if it detects a network disconnect or page error.  If your server goes down and the page becomes unresponsive, you should send a command to reload the page from Home Assistant or other platform on startup.  HA has a startup script you can run to send a MQTT/HTTP command to refresh the application page on startup.

**Comment by thanksmister** [comment]
Duplicate of https://github.com/thanksmister/wallpanel-android/issues/230
